### ISSUE TITLE
Confused about Training the DeepLab-V3+ 

### ISSUE THREAD

**Issue by xinyuaning** [qjadud1994/DRS#13]
Thank you for a great job!
  "In contrast, DeepLab-V3 does not require the COCO-pretrained weight due to the recent large memory GPUs and Synchronized BatchNorm" confused me. Is deeplab-V3+ is trained from scratch without any pre-trained weight?
Then I have a question about the link. The link of Deeplab-V3+ with ResNet-101 is pre-trained weight? or it's the model to test and get the results, 71%.
<html>
<body>

Model | mIoU | mIoU + CRF | pretrained
-- | -- | -- | --
DeepLab-V2 with ResNet-101 | 69.4% | 70.4% | [link]
DeepLab-V3+ with ResNet-101 | 70.4% | 71.0% | [link]

</body>
</html>
I'm looking forward to hearing from you.

**Comment by Zhengyangnuaa** [comment]
same question.
But I try to validate these two models before any training, both of they get a miou result around 0.01.
It seems none of them get the pretrained network(if so, the miou should not be low like this)

**Comment by qjadud1994** [comment]
Sorry for the late reply.

_Is deeplab-V3+ is trained from scratch without any pre-trained weight?_
>> DeepLab-V3+ requires ImageNet pretrained weight.

_The link of Deeplab-V3+ with ResNet-101 is pre-trained weight? or it's the model to test and get the results, 71%._
>> The link is the final weight that produces 71% mIoU.
